### PR TITLE
Doc fix: SSL endpoints

### PR DESCRIPTION
**Pull request by vinaypyati**
configure endpoints with http+ssl instead tcp.

_Needs discussion_

### CODE REVIEW

File: Documentation/Books/Manual/Administration/Configuration/SSL.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ SSL Endpoints
 
 Given a hostname:
 
-`--server.endpoint tcp://hostname:port`
+`--server.endpoint http+ssl://hostname:port`
```
Comment (ObiWahn) [inline Documentation/Books/Manual/Administration/Configuration/SSL.md]: I think it should be `tcp+ssl`.

The arangod help says:
`--server.endpoint <string...>                               endpoint for client requests (e.g. 'http+tcp://127.0.0.1:8529', or 'vst+ssl://192.168.1.1:8529') (default: )`

I would really prefer `http / https / vst / vsts` but for some reason we have that syntax. Maybe other variations work as well but then we would need to update the help and the manpages.

Comment (ObiWahn) [reply Documentation/Books/Manual/Administration/Configuration/SSL.md]: devel feels a bit broken:
- `http+ssl` and `tcp` work
- `tcp+ssl` and `http` do not work

Comment (jsteemann) [reply Documentation/Books/Manual/Administration/Configuration/SSL.md]: It should just `ssl://`.

**Comment by vinaypyati** [comment]
Tested with TLSv1.2 on ArangoDB 3.3.9

`tcp+ssl://` and `http://` do not work.
`tcp://` works but does not open up secure end-point[can be accessed with `http://localhost:8529` only not with `https://localhost:8529`].
`http+ssl://` works.
just `ssl://` works.

**Pull request merged**